charts/nirmata/values.yaml

```diff
@@ -93,6 +96,9 @@ podSecurityContext: {}
 
 # -- Security context for the containers
 securityContext:
+  {{- if eq .Values.platform "openshift" }}
```

Comment:
Instead, add security context to the kyverno.helm.values in kyverno-cr of enterprise kyverno operator chart.